static/app/utils/customMeasurements/customMeasurements.tsx

```diff
@@ -0,0 +1,7 @@
+type CustomMeasurement = {
+  key: string;
+  name: string;
+  supportedFunctions: string[];
```

Comment:
If we call this `functions` instead of `supportedFunctions`, I think we can avoid defining `measurementsMetaResponse` and just use `Promise<CustomMeasurementCollection>`

It does lose out on the explicitness of defining the functions as "supported" but I'm wondering if you think it's fine to avoid needing another type for `fetchCustomMeasurements`

Reply:
I've updated the name to `functions`, i think the name is cleaner. We will still have to use `measurementsMetaResponse` though since `measurementsMetaResponse` doesnt really follow the same format even with the name change.